code/controllers/subsystem/mapping.dm

```diff
@@ -106,29 +115,38 @@ SUBSYSTEM_DEF(mapping)
 	if(last)
 		QDEL_NULL(loader)
 
-/datum/controller/subsystem/mapping/proc/CreateSpace(MaxZLevel)
-	while(world.maxz < MaxZLevel)
-		++world.maxz
-		CHECK_TICK
-
 #define INIT_ANNOUNCE(X) to_chat(world, "<span class='boldannounce'>[X]</span>"); log_world(X)
 /datum/controller/subsystem/mapping/proc/loadWorld()
 	//if any of these fail, something has gone horribly, HORRIBLY, wrong
 	var/list/FailedZs = list()
 
 	var/start_time = REALTIMEOFDAY
+	INIT_ANNOUNCE("Loading CentCom")
+	TryLoadZ("_maps/map_files/generic/CentCom.dmm", FailedZs)
+
+	//splash turfs are ready, send em in
+	for(var/mob/dead/new_player/NP in GLOB.player_list)
+		NP.SendToStartLocIfRequired()
+
+	INIT_ANNOUNCE("Loading Lavaland")
+	TryLoadZ("_maps/map_files/Mining/Lavaland.dmm", FailedZs)
```

Comment:
Shouldn't these be configs?

Reply:
I'll gladly do it if were ever get a mining level other than lavaland